_posts/en/0500-10-24-tm-admin_en.md

```diff
@@ -13,30 +13,34 @@ category: coordination
+-  [Create a project](/en/coordination/tm-admin/#create-a-mapping-project)
+    -  [Define mapping area](/en/coordination/tm-admin/#define-mapping-area)
+    -  [Define tasks](/en/coordination/tm-admin/#define-tasks)
+    -  [Project area trim](/en/coordination/tm-admin/#project-area-trim)
+    -  [Advanced: Define mapping project areas and tasks with imported geodata](/en/coordination/tm-admin/#advanced-define-mapping-project-areas-and-tasks-with-imported-geodata)
+-  [Edit a mapping project](/en/coordination/tm-admin/#edit-a-mapping-project)
+    -  [Description - the first thing users learn about your project](/en/coordination/tm-admin/#description)
+    -  [Instructions - what the users should do](/en/coordination/tm-admin/#instructions)
+    -  [Metadata - where the project belongs to](/en/coordination/tm-admin/#metadata)
+    -  [Priority Areas - where users should start mapping](/en/coordination/tm-admin/#priority-areas)
+    -  [Imagery - the basis for mapping](/en/coordination/tm-admin/#imagery)
+    -  [Permissions - who is allowed to map and validate](/en/coordination/tm-admin/#permissions)
+    -  [Helpful hints - how you should fill in all these fields](/en/coordination/tm-admin/#helpful-hints)
+    -  [Publish - do not forget or you never get work done](/en/coordination/tm-admin/#publish)
 
-The Tasking Manager is the essential tool for all kind of groups to coordinate mapping on OpenStreetMap. It helps you to conduct a mapathon, manage a humanitarian mapping initiatives, or you want to create mapping tasks for students. The application divides an area of interest into manageable geographic chunks, that can be completed rapidly and collaboratively. This approach allows the distribution of tasks to many individual mappers, while reducing possible editing conflicts. 
+The Tasking Manager is the essential tool for all kind of groups to coordinate mapping on OpenStreetMap. It helps you to conduct a mapathon, manage a humanitarian mapping initiative, or whenever you want to create mapping tasks for students. The application divides an area of interest into manageable geographic chunks, that can be completed rapidly and collaboratively. This approach allows the distribution of tasks to many individual mappers, while reducing possible editing conflicts. 
 
-The Software further supports mapping accuracy and data quality by providing a consistent set of instructions for your mappers (e.g. 'map all roads and buildings'). In short, the Tasking Manager is how you set up and direct the workflow for open mapping activities. This module describes the basic administration of the  Tasking Manager for successful mapping events.
+The Software further supports mapping accuracy and good data quality by providing a consistent set of instructions for your mappers (e.g. 'map all roads and buildings'). In short, the Tasking Manager is how you set up and direct the workflow for open mapping activities. 
```

Comment:
Suggestion: 
The Software encourages accurate and quality mapping data by providing a consistent set of instructions for your mappers (e.g. 'map all roads and buildings'). In short, the Tasking Manager is how you set up and direct the workflow for open mapping activities.